db/migrate/20160405072347_add_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnName < ActiveRecord::Migration
+  def change
+  	add_column :employees, :department, :string
+  end
+end
```

Comment:
deleted one, had the same id as previous one so had to create another lol, deleted the duplicate